src/main/external-resources/renderers/Google-Android-BubbleUPnP-MXPlayer.conf

```diff
@@ -96,67 +96,40 @@ UpnpDetailsSearch = BubbleUPnP Media Renderer
-Supported = f:mpeg2     v:mpeg2                                                                                 m:video/mpeg2
-Supported = f:mpegps    v:mpeg1|mpeg2|vc1                            a:ac3|dts|eac3|lpcm|mp3|mpa                m:video/mpeg
-Supported = f:mpegts    v:h264|h265|mpeg2|vc1                        a:aac|aac-he|ac3|eac3|dts|lpcm|mpa         m:video/vnd.dlna.mpeg-tts
-Supported = f:ogg       v:divx|theora                                a:vorbis                                   m:video/ogg
-Supported = f:rm        v:rm                                         a:aac|aac-he|ac3|cook|ralf                 m:application/vnd.rn-realmedia-vbr
-Supported = f:webm      v:vp8|vp9                                    a:vorbis                                   m:video/webm
-Supported = f:wmv       v:mp4|mpeg2|vc1|wmv                          a:adpcm|lpcm|mpa|wma                       m:video/x-ms-wmv
+Supported = f:3g2|3gp   v:h263|h264|mp4                              a:aac-lc|he-aac|amr|mp3                       m:video/3gpp
+Supported = f:avi|divx  v:cvid|divx|dv|h263|h264|mjpeg|mp4|vc1|wmv   a:aac-lc|adpcm|ac3|lpcm|mp3|mpa|wma           m:video/avi
+Supported = f:dv        v:dv                                         a:lpcm                                        m:video/mpeg
+Supported = f:flv       v:h264|sor|vp6                               a:aac-lc|he-aac|adpcm|mp3|lpcm                m:video/x-flv
+Supported = f:h265      v:h265                                       a:aac-lc|he-aac                               m:video/h265
+Supported = f:mkv       v:divx|h264|h265|mp4|mpeg2|vc1|vp8           a:aac-lc|he-aac|ac3|adpcm|dts|eac3|flac|lpcm|mp3|mpa|opus|tta|vorbis|wavpack|wma   m:video/x-matroska
+Supported = f:mov       v:divx|h263|h264|jpg|mp4|sor                 a:aac-lc|he-aac|adpcm|amr|lpcm|mp3|mpa|qdmc   m:video/quicktime
+Supported = f:mp4       v:divx|h264|jpg|mp4                          a:aac-lc|he-aac|ac3|amr|lpcm|mp3|mpa          m:video/mp4
+Supported = f:mpeg2     v:mpeg2                                                                                    m:video/mpeg2
+Supported = f:mpegps    v:mpeg1|mpeg2|vc1                            a:ac3|dts|eac3|lpcm|mp3|mpa                   m:video/mpeg
+Supported = f:mpegts    v:h264|h265|mpeg2|vc1                        a:aac-lc|he-aac|ac3|eac3|dts|lpcm|mpa         m:video/vnd.dlna.mpeg-tts
+Supported = f:ogg       v:divx|theora                                a:vorbis                                      m:video/ogg
+Supported = f:rm        v:rm                                         a:aac-lc|he-aac|ac3|cook|ralf                 m:application/vnd.rn-realmedia-vbr
+Supported = f:webm      v:vp8|vp9                                    a:vorbis                                      m:video/webm
+Supported = f:wmv       v:mp4|mpeg2|vc1|wmv                          a:adpcm|lpcm|mpa|wma                          m:video/x-ms-wmv
```

Comment:
@ler0y They are planned to: https://github.com/UniversalMediaServer/UniversalMediaServer/compare/Sami32-patch-6 :wink: